packages/link-modules/src/definitions/cart-region.ts

```diff
@@ -0,0 +1,28 @@
+import { Modules } from "@medusajs/modules-sdk"
+import { ModuleJoinerConfig } from "@medusajs/types"
+
+export const CartRegion: ModuleJoinerConfig = {
+  isLink: true,
+  isReadOnlyLink: true,
+  extends: [
+    {
+      serviceName: Modules.CART,
+      relationship: {
+        serviceName: Modules.REGION,
+        primaryKey: "id",
+        foreignKey: "region_id",
+        alias: "region",
+      },
+    },
+    {
+      serviceName: Modules.REGION,
+      relationship: {
+        serviceName: Modules.CART,
+        primaryKey: "id",
+        foreignKey: "cart_id",
+        alias: "carts",
+        isList: true,
+      },
```

Comment:
```suggestion
      serviceName: Modules.REGION,
      relationship: {
        serviceName: Modules.CART,
        primaryKey: "region_id",
        foreignKey: "id",
        alias: "carts",
        isList: true,
      },
```

Reply:
Maybe we can cover these configs on the tests, querying the new properties.

Reply:
Something like this: https://github.com/medusajs/medusa/pull/6392/commits/2225f157fe49fdd428128dc778779d38164d15d9